src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor

```diff
@@ -46,6 +46,19 @@
             </div>
         </div>
 
+        @if (TelemetryService is { IsTelemetryInitialized: true, IsTelemetrySupported: true })
```

Comment:
Shouldn't `IsTelemetrySupported` be enough here? I think by this point if telemetry is supported then it would always be initialized.